### DISCUSSION TITLE
Feature Request: Baseline Update - List and CherryPick pending commits

### DISCUSSION THREAD

**Discussion by c9mb** [umbraco/Umbraco.Cloud.Issues#789] [Ideas]
### Issue description

I have often encountered the scenario where I have a bunch of commits pending on the Baseline-Master that I do NOT yet want to push to some or all Baseline-Child projects, but do have some critical commits, such as a bug-fix or even some Umbraco-Created commits, that I DO need to push to selected child projects. 

In fact I encountered this situation a few days ago where the Master received updates for Umbraco-Forms and Umbraco-Deploy but a large number of the child projects did not (for unknown reasons) and I'd like to push those Umbraco commits without pushing others.

It would be great if I could:

1. See a list of the pending commits on the Baseline-Master that would apply to selected Baseline-Children.
2. CherryPick commits to push from the Baseline-Master to the selected Baseline-Children.

**Comment by sajumb** [comment]
Thanks for the feature request. This is noted and will be considered in the current process where we discover/define which deploy features to be included in the cloud portal.

**Comment by miguelcrpinto** [comment]
Any updates/news on this topic?

**Comment by sajumb** [comment]
We currently do not have an update on this one. But we are conceptually working on various new features in Umbraco Cloud somehow related to this one, which we will share more insight about later this spring/early summer.
